database/java/2015/6420.yaml

```diff
@@ -0,0 +1,42 @@
+---
+cve: CVE-2015-6420
+title: CVE in commons-collections:commons-collections
+description: >
+    Serialized-object interfaces in certain Cisco Collaboration and Social Media; Endpoint Clients and Client Software; Network Application, Service, and Acceleration; Network and Content Security Devices; Network Management and Provisioning; Routing and Switching - Enterprise and Service Provider; Unified Computing; Voice and Unified Communications Devices; Video, Streaming, TelePresence, and Transcoding Devices; Wireless; and Cisco Hosted Services products allow remote attackers to execute arbitrary commands via a crafted serialized Java object, related to the Apache Commons Collections (ACC) library.
+cvss_v2: 7.5
+references:
+    - http://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-20151209-java-deserialization
+    - http://www.securityfocus.com/bid/78872
+    - https://h20566.www2.hpe.com/portal/site/hpsc/public/kb/docDisplay?docId=emr_na-c05376917
+    - https://h20566.www2.hpe.com/portal/site/hpsc/public/kb/docDisplay?docId=emr_na-c05390722
+    - https://www.kb.cert.org/vuls/id/576313
+    - https://www.tenable.com/security/research/tra-2017-14
+    - https://www.tenable.com/security/research/tra-2017-23
+
+affected:
+    - groupId: commons-collections
+      artifactId: commons-collections
+      version:
+        - "== 3.2.1 & == 4.0"
```

Comment:
Not a valid syntax. Should be:
```
- "<=3.2.1,3.0"
- "==4.0"
```

Reply:
Similarly in `fixedin`

Reply:
@msrb as per the ref i could see that  versions 3.2.1 and 4.0 are only affected,any specific reason or ref to update as <=3.2.1,3.0  ?